### ISSUE TITLE
[AppV2] Opening flows tab is firing the same flow over and over

### ISSUE THREAD

**Issue by SailingDuck** [athombv/homey#1873]
**What is your Homey version (Settings → System)?**

1.5.3 / Android preview 0.5.0

**What did you try to achieve?**

*Please include detailed steps how we can reproduce the issue*

Just opening the flows tab in the new Android app

**What was the result?**

Flow fires continuously, noticable because homey is talking in it.

**What did you expect as result?**

No flow execution

**Any other remarks**

Guessing it has something to do with the card 'Deze flow is gestart' in the first column. Then column casts a webpage and a 'Zeg iets' card. Second column is empty.

**Please attach a Homey System log (Settings -> System -> Send Log)**

Let me know if needed.

**Comment by Dijker** [comment]
I have also seen a flow triggering the moment I was exploring the app yesterday evening. 
Had to close  the app to prevent "roar in the keet, a low waf and angry girlfriend  "

**Comment by SailingDuck** [comment]
Also a flow is started card in the first column?

**Comment by Dijker** [comment]
As the girlfriend  left the house I tested again this morning:

I think I closed the app within 30 seconds. 
Analyzing and Viewing the logs, 
~ 150 flows triggered from switching lights, Homey talking, sending emails etc.  during 65 seconds
I guess all harm is done by the Flows with a "This flow is started" trigger card. 
and probably others flows are triggered by them. 
~10 times all lights switched on and off (randomized)
I received 8 emails from Homey 
and have had ~28 Push notifications. 

Two time the Screen of the App reported Homey Disconnected...

The App reports "flow_not_enbled" and disconnected 2 times from Homey

![image](https://user-images.githubusercontent.com/12412734/31424816-a4f0cf60-ae5c-11e7-8ef8-4eec26c937ac.png)
